src/content/app/tools/blast/state/blast-api/blastApiSlice.ts

```diff
@@ -62,9 +64,15 @@ const blastApiSlice = restApiSlice.injectEndpoints({
       BlastSubmissionPayload
     >({
       query(payload) {
+        // merge the sequence and header using tofasta
```

Comment:
This comment describes _what_ the code below is doing (which can be read from the code itself), rather than _why_ this is being done. Maybe replace with something like "The backend accepts sequences as FASTA strings".